nft-rfc-008.md

```diff
@@ -55,117 +55,111 @@ content of a verifiable claim, to determine whether the claim meets specific acc
-of claims the Claim Verification Agent will be evaluating. This Verifiable Credential establishes the validity of the verification 
-claims made by the Claim Verification Agent. 
-
-In the case that the Claim Verification Agent is also the recognized governing body which oversees the evaluation of claims of the 
-type being made VC4 may not be necessary. 
+## Verification Process
 
-### VC5
-The final Verifiable Credential created in the process. This is tokenized in the minting of the NFT IID which is returned to the 
-Claim Producer via a retrieval capability and contains, at least by reference, the set of Verifiable Credentials utilized in the 
-process of verifying the initial claims made by the Claim Producer. It must provide the complete set of 
-Verifiable Credentials (V1, V2, V3 and V4) so the full chain of verification can be inspected by third parties interested in the 
-final NFT IID can inspect the origins of the NFT.
+1. The final issuing authority of the NFT begins by specifying a schema template for the NFT they will be issuing.
+   This schema template will define both set of criteria which a Claim Issuer has the burden of satisfying for the 
+   NFT issuing authority to accept the claim being made as true and a set of criteria that must be met for the NFT
+   issuing authority to receive a minting capability. The criteria to recieve the minting capability from the NFT issuing
+   authority and the criteria for the acceptance of a claim or set of claims can and likely will be different. 
+   If the criteria, as definited in the schema template, are not met to the satisfaction of the NFT issuing authority 
+   they can revoke the minting capability of the Claim Issuer and/or burn any minted NFT(s).
+   Some example criteria that could be specified:
+   * Claim Issuer must provide a certification from a certifying agency recognised by the NFT issuing authority
+   * Claim Issuer must have their Claim validated by a recognized evaluating agency
+   * Claim Issuer must provide evidence as referrenced by the Claim Evaluation
+   * Claim must be submitted within a certain timeframe of the claim issuing date
+   * etc
+2. Once the NFT issuing authoriy has specified their schema a Claim Issuer can now evaluate the published schema and 
+   begin the process of collecting any credentials they may need to satisfy the criteria set forth in the schema. For
+   the Claim Issuer this may include:
+   * Contacting a recognized certification agency who can inspect their infrastructure and provide an inspection certificate.
+   * Finding a certified Claim Verification Agent to evaluate their claims
+   * Ensure the can provide the evidence needed by the Claim Verification Agent
+   * etc
+3. Now that the Claim Issuer has collected the necissary pieces to fullfil the NFT issuing authority's criteria they can now
+   provide an initial set of proof of criteria fulfillment to the NFT issuing authority, void of any claim, and receive
```

Comment:
Rather than 'void of any claim', we should introduce the `caveats` terminology from the Z-Cap spec